contactPage/contactPage.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <title>Joseph:Home</title>
+</head>
+<body>
+    <nav>
+        <div class="navContainer">
+            <div class="logoName">
+                JaHolt
+            </div>
+            <ul>
+                <a href="contactPage.html"><li>CONTACT</li></a>
+                <a href="../portfolioPage/portfolioPage.html"><li>PORTFOLIO</li></a>
+                <a href="../aboutMePage/aboutMePage.html"><li>ABOUT</li></a>
+                <a href="../index.html"><li>HOME</li></a>
+            </ul>
+            <div class="navAlignBox">
+            </div>
+        </div>
+    </nav>
+    <main>
+        <section class="titleContainer">
+            <div class="titleBox">
+                <img src="../css/pics/lightbulb.png" alt="lightbulb logo" />
+                <h1>Contact</h1>
+            </div>
+        </section>
+        <section class="subcategoryContainer">
+            <div class="contactDetailsBox">
+                <h3>Contact Details:</h3>
+                <p>Email: holt.ja.halfways@gmail.com</p>
+                <p class="telephoneText">Telephone: <h6>000000000000000000000</h6></p>
```

Comment:
Putting an h6 inside of a p tag is bad form and people will make fun of you. You could try replacing this with a strong tag, which is an inline element and fits spec.